packages/constants/src/index.ts

```diff
@@ -409,3 +409,22 @@ export enum MASK_TYPES {
     STENCIL = 2,
     SPRITE = 3,
 }
+
+/**
+ * Constants for multi-sampling antialiasing
```

Comment:
I would like really more documentation about this feature being _experimental_ and that there's a significant performance cost paid for using it. Also, it's not super obvious, but a code example usage setting `renderer.framebuffer.multisample`.

Reply:
linked it to Framebuffer#multisample. Don't want to ctrl+C everything.